cdap-common/src/main/java/co/cask/cdap/common/conf/KafkaConstants.java

```diff
@@ -26,14 +26,20 @@
    */
   public static final class ConfigKeys {
 
-    public static final String PORT_CONFIG = "kafka.bind.port";
-    public static final String NUM_PARTITIONS_CONFIG = "kafka.num.partitions";
-    public static final String LOG_DIR_CONFIG = "kafka.log.dir";
-    public static final String HOSTNAME_CONFIG = "kafka.bind.address";
-    public static final String ZOOKEEPER_NAMESPACE_CONFIG = "kafka.zookeeper.namespace";
-    public static final String REPLICATION_FACTOR = "kafka.default.replication.factor";
```

Comment:
`kafka.bind.port` and `kafka.default.replication.factor` do not have deprecated versions (unless they are elsewhere and I am just missing them, or unless they actually aren't required):
```
@Deprecated
public static final String PORT_CONFIG_DEPRECATED = "kafka.bind.port";
@Deprecated
public static final String REPLICATION_FACTOR_DEPRECATED = "kafka.default.replication.factor";
```
I think it would be good to keep these in the same order as the original, as it makes it easier to check...